### DISCUSSION TITLE
No parallax background or sound

### DISCUSSION THREAD

**Discussion by caidenmccarthy** [Hexeption/MCP-Reborn#167] [Q&A] [answered]
I followed the steps up until running the game but when i did i didnt see a background and i didnt hear any sounds. i also cannot find the assets folder that has all of the art of the game

**Comment by binchiling100** [comment]
My sounds do play but i also cant find assets

**Comment by EndermaxOff** [comment]
Me too, i just find .class document but no one is .png or .ogg

**Comment by EndermaxOff** [comment]
I find .jar wihout png on build/libs sadly...

**Comment by Hexeption** [comment, marked as answer]
Set working directory to run folder

**Reply by dieselfox1** [reply]
same problem, my run folder is set as my working directory, optifine nor my client can't read font data i have located in /assets/minecraft and i have optifine assets thrown in and it seems it can't find it.

**Reply by dieselfox1** [reply]
my sound and lwjgl libs are loaded and working fine, ran copyassets, minecraft won't even load mojang screen just white blank screen before it crashes because it can't locate assets folder and my client font data i have located there
